Ports/cavestory/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="cavestory"
+version="2.6.5-1"
+files="https://github.com/nxengine/nxengine-evo/archive/refs/heads/master.zip master.zip d013acbf399938b5d3d10fd4d37817b630559e2b24b90576cde6f88d445ea150"
+auth_type='sha256'
+depends=(
+  "gcc"
+  "cmake"
+  "SDL2"
+  "SDL2_ttf"
+  "libpng"
+  "SDL2_mixer"
+  "libjpeg"
+  "SDL2_image"
```

Comment:
1. Please use 4 spaces as indentation
2. All these entries should use single quotes
3. Please put them in alphabetical order